This patch avoid using pthread when we build Windows binary.
Windows is non-POSIX OS. Therefore, we should use thread of windows native instead pthread.

In addition, this patch is temporaliy.
Because this modification will include in the next version.
We will remove this patch when we will release the next version.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b1e3c98d..54e5cfc69 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -337,7 +337,11 @@ check_struct_has_member("struct tm" "tm_gmtoff" "time.h"
                         HAVE_STRUCT_TM_TM_GMTOFF)

 find_package(Threads REQUIRED)
-set(HAVE_PTHREAD_H ${CMAKE_USE_PTHREADS_INIT})
+if(WIN32)
+  set(HAVE_PTHREAD_H FALSE)
+else()
+  set(HAVE_PTHREAD_H ${CMAKE_USE_PTHREADS_INIT})
+endif()

 option(GRN_WITH_NFKC "use NFKC based UTF8 normalization." ON)
